packages/api/src/task/scheduler.ts

```diff
@@ -130,7 +130,7 @@ export class TaskScheduler {
     let assetSpec: Asset;
     switch (task.type) {
       case "import":
-      case "upload":
+      case "upload" || "clip":
```

Comment:
This will not work, `"upload" || "clip"` evaluates to `"upload"`. You have to do the same thing as the line above.

Reply:
Oh right! thank you